scheds/rust/scx_rustland/src/bpf.rs

```diff
@@ -226,6 +226,24 @@ impl<'a> BpfScheduler<'a> {
         }
     }
 
+    // Override the default scheduler time slice (in us).
+    #[allow(dead_code)]
+    pub fn set_slice_us(&mut self, slice_us: u64) {
+        self.skel.bss_mut().sched_slice_ns = slice_us * 1000;
+    }
+
+    // Get current value of time slice (slice_ns).
+    #[allow(dead_code)]
+    pub fn get_slice_us(&mut self) -> u64 {
+        let slice_ns = self.skel.bss().sched_slice_ns;
```

Comment:
The naming is confusing. Maybe `dynamic_slice_ns` or `effective_slice_ns` would be better?

Reply:
I like `effective_slice_ns`, it's a lot more clear, I'll go with that. Thanks!